executor/common_linux.h

```diff
@@ -1557,6 +1557,86 @@ static long syz_io_uring_submit(volatile long a0, volatile long a1, volatile lon
 
 #endif
 
+#if SYZ_EXECUTOR || __NR_syz_usbip_server_init
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+static int proc_id;
+static int port_num;
+
+static long syz_usbip_server_init(void)
+{
+	int fd;
+	int rc;
+	int socket_pair[2];
+	char buffer[100];
+	int server_fd, client_fd;
+	char path[100];
+
+	rc = socketpair(AF_UNIX, SOCK_STREAM, 0, socket_pair);
+	if (rc < 0) {
+		fail("syz_usbip_server_init : socketpair failed: %d\n", rc);
+	}
+
+	client_fd = socket_pair[0];
+	server_fd = socket_pair[1];
+
+        int available_port_num = __atomic_fetch_add(&port_num, 1, __ATOMIC_RELAXED);
+	if (available_port_num  > VHCI_HC_PORTS){
+            fail("syz_usbip_server_init : no more available port for : %d\n", available_port_num);
+        }
+
+	sprintf(buffer, "%d %d %s", available_port_num, client_fd, "65537 3");
```

Comment:
Pass `speed flags[usb_device_speed]` to `syz_usbip_server_init` as 1st arg  and use it here.

Also use `0` for `devid` and add a comment that it doesn't matter.